### PR TITLE
KOGITO-2716: Move kogito-test-utils to kogito-runtimes

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA: https://issues.redhat.com/browse/KOGITO-2716
Description: Move kogito-test-utils from kogito-examples to kogito-runtimes.

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by Sgitario** [comment]
jenkins please retest this

**Comment by Sgitario** [comment]
I need to increase the code coverage. WIP.

**Comment by Sgitario** [comment]
jenkins please retest this

File: addons/persistence/infinispan-quarkus-health-addon/pom.xml [hunk 1]
```diff
@@ -46,12 +46,14 @@
...
-      <artifactId>testcontainers</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
+	  <groupId>org.kie.kogito</groupId>
+	  <artifactId>kogito-test-utils</artifactId>
+	  <scope>test</scope>
+	</dependency>
+	
+	<dependency>
```
Comment (cristianonicolai) [inline addons/persistence/infinispan-quarkus-health-addon/pom.xml:56]: format a bit off?

File: kogito-test-utils/pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+  	<dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+      <scope>compile</scope>
```
Comment (cristianonicolai) [inline kogito-test-utils/pom.xml:41]: no need to declare `compile` scope, as it is the default anyway.

File: kogito-test-utils/pom.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${version.springboot}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+  	<dependency>
```
Comment (cristianonicolai) [inline kogito-test-utils/pom.xml:38]: please check format

File: kogito-test-utils/pom.xml (hunk 2 above)
Comment (Sgitario) [reply kogito-test-utils/pom.xml:41]: The testcontainers dependency is declared in the runtimes parent using the scope test, so I need this scope here.

File: kogito-test-utils/pom.xml (hunk 3 above)
Comment (Sgitario) [reply kogito-test-utils/pom.xml]: Fixed

File: addons/persistence/infinispan-quarkus-health-addon/pom.xml (hunk 1 above)
Comment (Sgitario) [reply addons/persistence/infinispan-quarkus-health-addon/pom.xml:56]: Fixed

File: kogito-test-utils/README.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
+# Common Utils For Kogito
```
Comment (evacchi) [inline kogito-test-utils/README.md:1]:
```suggestion
# Common Testing Utilities For Kogito
```

**Comment by tiagodolphine** [comment]
Hi @Sgitario what is missing to get this PR merged? I'm working on the integrations tests for job service into kogito-apps, and I'm using this test-utils and need to include more things, so it would be nice if we get this PR into master.
Thanks.

**Comment by tiagodolphine** [comment]
@Sgitario anything missing on the PR to get it merged?

**Comment by Sgitario** [comment]
@tiagodolphine, it's missing the review from @jstastny-cz .

File: addons/persistence/infinispan-quarkus-health-addon/pom.xml (hunk 1 above)
Comment (jstastny-cz) [reply addons/persistence/infinispan-quarkus-health-addon/pom.xml:56]: Seems like tab instead of 4 spaces.

Comment (Sgitario) [reply addons/persistence/infinispan-quarkus-health-addon/pom.xml]: I missed to format the last dependency, fixed now. Thanks for spotting!

**Pull request merged**